servicecontrol/logging.md

```diff
@@ -10,18 +10,71 @@ redirects:
 
 ### Logging
 
-Instances of the ServiceControl service write logging information and failed message imports to the file system.  
-Before v1.9 the logging level by default was set to `Info`, however this level can be quite verbose, so from v1.9 the default logging level is now `Warn`, this level is also configurable by adding the following to the configuration file:
+Instances of the ServiceControl service write logging information and failed message import stack traces to the file system.  
```

Comment:
It reads to me that both loggin information and failed message imports are written to flie system ?